### DISCUSSION TITLE
Consider moving the precompiles to the "script" directory

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/evm-monorepo#662] [Ideas]
Refs:

- https://github.com/sablier-labs/v2-periphery/pull/91#issuecomment-1580391265
- https://github.com/sablier-labs/v2-periphery/discussions/93

**Comment by smol-ninja** [comment]
Since precompiles have been moved to their own directory, https://github.com/sablier-labs/v2-core/pull/841, I am closing this discussion.
